schema/spacewalk/common/tables/rhnErrata.sql

```diff
@@ -36,6 +36,7 @@ CREATE TABLE rhnErrata
     synopsis          VARCHAR(4000) NOT NULL,
     topic             VARCHAR(4000),
     solution          VARCHAR(4000) NOT NULL,
+    rights            VARCHAR(100),
```

Comment:
I wonder if "100" is enough. Did you check what is used out there?

Reply:
Yes, all that I saw is a variation of "Copyright <year> <company>". The longest I tested is from RockyLinux "Copyright (C) 2021 Rocky Enterprise Software Foundation" which is 55 characters. But you have a point, maybe I can bump it to 200 to be extra safe.

Reply:
Ok, then 100 should be good enough